### PR TITLE
Rules for Bonus Round in SAUVC 2024

### PR DESCRIPTION
**Pull request by notthetup**
Rendered PDF : [rulebook.pdf](https://github.com/sauvc/rulebook/blob/sauvc-2024-bonus/rulebook.pdf)

### CODE REVIEW

**Review by mchitre — APPROVED** [review]

File: bonus.md
Comment (harivnkochi) [inline bonus.md:8]: Best not to use any TCOMS image, lets just keep a blank image here as placeholder until we get approval from TCOMS to use an image.

File: bonus.md
Comment (harivnkochi) [inline bonus.md]: Typo: Ocean Basin

File: bonus.md
Comment (harivnkochi) [inline bonus.md:29]: We can only enter the 'basin' itself when it is raised, and I imagine the floor would be lowered that day which means no one will be allowed to enter the basin (except divers maybe).

I think the best wording here would be:
"Since the basin is a research facility, and the area is not typically open to the public, only a
small predetermined number of the participants/viewers will be allowed into the facility to watch the attempt from the viewing gallery. Members of the team doing the attempt will have to operate at the AUV Deployment/Retrieval corner under the supervision of SAUVC committee members. SAUVC Committee members, judges and TCOMS staff will be located in the subcarriage in the facility which will be positioned above the gate, providing a direct view of the gate. After a successful round/abort, the AUV will be retrieved by divers.

Details on the number of participants allowed to enter the viewing gallery will be released at a later date."

File: bonus.md
Comment (harivnkochi) [inline bonus.md]: The sentence "The teams will be given a time slot to attempt the bonus round." can be deleted, as the next sentence covers it.

Also, typo: retreival -> retrieval

**Review by harivnkochi — COMMENTED** [review]
A bunch of comments on the sentences in the bonus round, here and there. Best to keep a blank image for the TCOMS basin until we get clearance from them to use any image.

**Review by harivnkochi — APPROVED** [review]

**Pull request merged**
